arch/arm/configs/mako_defconfig

```diff
@@ -230,6 +230,7 @@ CONFIG_NET_EMATCH_META=y
 CONFIG_NET_EMATCH_TEXT=y
 CONFIG_NET_CLS_ACT=y
 CONFIG_BT=y
+CONFIG_BT_HCISMD=y
```

Comment:
We keep tradition to add our mods to the bottom of the defconfig, after the #Mer modifications comment